packages/experiments/src/components/Chiclet/ChicletCard.tsx

```diff
@@ -0,0 +1,148 @@
+        className={
+          css('ms-ChicletCard', className) }
+      >
+        <div
+          className={ mergeStyles(this._classNames.preview) }
+        >
+          { image ?
+            preview :
+            (<Image
+              src={ TestImages.documentPreviewTwo }
+              role='presentation'
+              alt=''
+            />)
+          }
+        </div>
+        <div
+          className={ mergeStyles(this._classNames.info) }
+        >
+          <div
+            className={ mergeStyles(this._classNames.title) }
+          >
+            { title ? title : "Placeholder" }
+          </div>
+          <div
+            className={ mergeStyles(this._classNames.link) }
+          >
+            { url ? url : "https://onedrive.com/files/v-lygi/39192908430" }
+          </div>
+          { actions ? this._renderFooter(actions) : (null) }
+        </div>
+      </div>
+    );
+  }
+
+  private _renderPreviewImage(imageUrl?: string, imageHeight?: string, imageWidth?: string, ogType?: string): React.ReactElement<React.HTMLAttributes<HTMLDivElement>> {
+    const image = (
+      <Image
+        width={ imageWidth }
+        height={ imageHeight }
+        src={ TestImages.documentPreview }
+        role='presentation'
+        alt=''
+      />
+    );
+
+    let icon;
+    switch (ogType) {
+      case "word":
+        icon = <Icon className={ mergeStyles(this._classNames.icon) } iconName='WordDocument' />;
+        break;
+      case "powerpoint":
+        icon = <Icon className={ mergeStyles(this._classNames.icon) } iconName='PowerPointDocument' />;
+        break;
+      case "excel":
+        icon = <Icon className={ mergeStyles(this._classNames.icon) } iconName='ExcelDocument' />;
+        break;
+    }
+
+    return (
+      <div>
+        { image }
+        { icon }
+      </div>
+    );
+  }
+
+  private _renderFooter(actions: IButtonProps[]): React.ReactElement<React.HTMLAttributes<HTMLDivElement>> {
+    return (
+      <div className={ mergeStyles(this._classNames.actions) }>
+        { actions && actions.map((action, index) => {
+          return (
+            <div className={ mergeStyles(this._classNames.action) } key={ index }>
+              <IconButton { ...action } />
+            </div>
+          );
+        }) }
+      </div>
+    );
+  }
+
+  private _onClick = (ev: React.MouseEvent<HTMLElement>): void => {
+    this._onAction(ev);
+  }
+
+  private _onKeyDown = (ev: React.KeyboardEvent<HTMLElement>): void => {
+    if (ev.which === KeyCodes.enter || ev.which === KeyCodes.space) {
+      this._onAction(ev);
+    }
+  }
+
+  private _onAction = (ev: React.SyntheticEvent<HTMLElement>): void => {
+    const { onClick, onClickHref } = this.props;
+
+    if (onClick) {
+      onClick(ev);
+    } else if (!onClick && onClickHref) {
+      // If no onClick Function was provided and we do have an onClickHref, redirect to the onClickHref
+      window.location.href = onClickHref;
```

Comment:
If I hold `CTRL` or `CMD` and click does it open the link in a new tab? That would be expected browser behavior so I'd check that scenario.

Reply:
Yes it does!

Reply:
Good point! I'll have to change that. For now though, I'm going to wait to see what exactly we're expecting from onClick for Chiclets before implementing further. I'll keep this as a placeholder for now.